### PR TITLE
Document exclude potential breaking change

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test RFM

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 <!-- Do not add breaking changes or deprecations here! Add them to the upgrade guide instead. --> 
 
-<!-- 
+<!-- `
```
Comment (tresat) [inline subprojects/docs/src/docs/release/notes.md:34]: Is this backtick intended?

**Review by tresat — APPROVED** [review]

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:34]: Nope, must have been a typo, removing.

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**